test/test-app/src/index.ts

```diff
@@ -1,6 +1,13 @@
 import { AutoRouter } from "itty-router"
 import { headersTest, health, kvTest, kvTestUint8Array, outboundHttp, statusTest, stream, streamTest, testFunctionality } from "./test";
 
+// Verify top level init location
+if (globalThis.location.href != "http://foo.bar/") {
+    throw new Error(`Expected top-level init location to be "top-level", got "${globalThis.location}"`);
+} else {
+    console.log("Top-level init location verified:", globalThis.location.href);
```

Comment:
Might as well be consistent with the use of strings vs templates:
```suggestion
    console.log(`Top-level init location verified: ${globalThis.location.href}`);
```